ballerina/Package.md

```diff
@@ -0,0 +1,13 @@
+## Package overview
+
+This package includes Ballerina `persist` Tooling, which provides functionality to store and query data conveniently through a data model instead of SQL query language.
```

Comment:
```suggestion
This package includes Ballerina `persist` support, which provides functionality to store and query data from Google Sheets conveniently through a data model instead of using Google Sheets APIs.
```